composer.json

```
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "^7.0"
+        "php": ">=7.0"
```

I don't want to support php@12 already ;)
```suggestion
        "php": "^7.0 || ^8.0"
```